src/helpers/featureFlag.js

```diff
@@ -0,0 +1,5 @@
+const featureIsEnabled = (feature) => {
+  return process.env[feature] === "yes" ? true : false;
```

Comment:
```suggestion
  return process.env[feature] === "yes";
```
`===` already produces a boolean so the `?` is unnecessary

Reply:
lool monday brain 🤦‍♀️